media/static-bundles.json

```diff
@@ -1650,9 +1650,16 @@
     },
     {
       "files": [
-        "js/base/dnt-helper.js"
+        "js/careers/base.js"
       ],
-      "name": "careers-dnt-helper"
```

Comment:
Has the DNT helper gone away entirely?

Reply:
It's already included in the base template bundle, so no need to bundle it again separately any more :)